typeguard/__init__.py

```diff
@@ -26,6 +26,11 @@
 except ImportError:
     Literal = TypedDict = None
 
+try:
+    from typing_extensions import Literal as BPLiteral
+except ModuleNotFoundError:
```

Comment:
How did you get the tests to pass locally on Python 3.5? ModuleNotFoundError was only introduced in 3.6.

Reply:
Heh, tested with 3.8, 3.7, and 3.6. I didn't have 3.5 and the CI tests pass because `typing_extensions` is indeed installed. Would you recommend `except ImportError` then?

Reply:
Right, of course. Yes, use `ImportError` here please.